Composer/packages/client/src/pages/publish/Notifications.tsx

```diff
@@ -56,29 +58,121 @@ export const getPublishedNotificationCardProps = (item: BotStatus): CardProps =>
             name: item.name,
             publishTarget: item.publishTarget,
           }),
-    type: 'pending',
+    type: item.status === 200 ? 'success' : 'error',
     onRenderCardContent: (props) => (
       <div css={cardContent}>
         <Icon css={infoType} iconName="CloudUpload" />
-        <Icon css={statusIconStyle} iconName={item.status === 200 ? 'SkypeCircleCheck' : 'CircleAdditionSolid'} />
+        <Icon css={statusIconStyle} iconName={item.status === 200 ? 'Completed' : 'CircleAdditionSolid'} />
         <div css={cardDetail}>
           <div css={cardDescription}>{props.description}</div>
         </div>
       </div>
     ),
   };
 };
-export const getPendingNotificationCardProps = (items: BotStatus[]): CardProps => {
+
+export const getSkillPublishedNotificationCardProps = (item: BotStatus, url?: string): CardProps => {
+  const skillCardContent = css`
+    display: flex;
+    padding: 0 16px 16px 12px;
+    min-height: 64px;
+  `;
+  const statusIconStyle = css({
+    margin: '12px 0 0 -1px',
+    width: '12px',
+    height: '12px',
+    fontSize: '12px',
+    color: '#D92525',
```

Comment:
Can we use the closest matching colors from Shared, Communication?

Reply:
Same for all colors in this file

Reply:
Sure

Reply:
@alanlong9278 I didn't mean to look for the exact hex code. I meant the closest matching color. for ex SharedColors.red10 is what we use throughout the project for #D92525. When we change the theme for Composer in the future these colors would adapt to the change.

Reply:
Sure